app/api/tickets.py

```diff
@@ -182,6 +182,11 @@ def before_update_object(self, ticket, data, view_kwargs):
                 raise UnprocessableEntity(
                     {'event_id': ticket.event.id}, "Event having paid ticket must have a payment method")
 
+        if ticket.deleted_at != data.get('deleted_at'):
```

Comment:
This will be true when restoring tickets from deleted as well

Reply:
So should I remove this? and just add the line for append?

Reply:
We should add a check when data.get('deleted_at') is null correct?

Reply:
Check if there's a feature to restore deleted tickets in admin panel. If there's not, then it won't need to be changed

Reply:
No, there's nothing as such @iamareebjamal